packages/pangea-sdk/pangea/services/audit/audit.py

```diff
@@ -2,14 +2,13 @@
 # Author: Pangea Cyber Corporation
 import datetime
 import json
-from typing import Dict, Optional, Union
+from typing import Any, Dict, Optional, Union
```

Comment:
where is this needed?

Reply:
@govind-salinas-pangea here in `event: Dict[str, Any],`